scripts/ci/update_release.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python3
+
+import json
+import sys
+import yaml
+from typing import Dict
+
+RELATIVE_PATH_TO_MANAGER_YAML = "config/manager/manager.yaml"
+RELATIVE_PATH_TO_OPENSHIFT_MANAGER_YAML = "deploy/openshift/operator_openshift.yaml"
+
+
+def _load_yaml_file(path: str) -> Dict:
+    with open(path, "r") as f:
+        return yaml.safe_load(f.read())
+
+
+def _dump_yaml(operator: Dict, path: str) -> None:
+    with open(path, "w+") as f:
+        return yaml.dump(operator, f)
+
+
+def update_and_write_file(path: str) -> None:
+    release = _load_release()
+    yaml_file = _load_yaml_file(path)
+    _update_operator_deployment(yaml_file, release)
+    _dump_yaml(yaml_file, path)
+
+
+def _load_release() -> Dict:
+    with open("release.json", "r") as f:
+        return json.loads(f.read())
+
+
+def _replace_tag(image: str, new_tag: str) -> str:
+    split_image = image.split(":")
+    return f"{split_image[0]}:{new_tag}"
```

Comment:
please avoid Python expressions in f-strings. I think it is better to do:

`split_image[0] + ":" + new_tag` (it can't get easier to read than this)